Scripts/post_build.ps1

```diff
@@ -139,12 +126,10 @@ function Main {
         $o = "$p\Output\Packages"
         Validate-Directory $o
         # making version static as multiple versions can exist in the nuget folder and in the case a breaking change is introduced.
-        New-Alias Nuget $env:USERPROFILE\.nuget\packages\NuGet.CommandLine\5.4.0\tools\NuGet.exe -Force
```

Comment:
Nuget comandline is needed for packing and releasify with Squirrel

Reply:
ok I can see AppVeyor has built the packages correctly. 
![image](https://user-images.githubusercontent.com/26427004/81549975-f0233880-93c2-11ea-9f2e-e1290009b4a8.png)

Are they self contained?

Reply:
* `NuGet.CommandLine` is an outdated way of using nuget, AppVeyor has it as a CLI tool and your dev environment most likely has it as well.
* self-contained executables are runtime-specific.... how will we deal with that? Will we offer different exe downloads [for every target platform](https://docs.microsoft.com/en-us/dotnet/core/rid-catalog)? What about Squirrel and update artifacts?
* The zip file is just the release folder compressed. Wox did that on every build, I saw no reason to remove it...

Reply:
> * `NuGet.CommandLine` is an outdated way of using nuget, AppVeyor has it as a CLI tool and your dev environment most likely has it as well.

OK fair enough.

> * self-contained executables are runtime-specific.... how will we deal with that? Will we offer different exe downloads [for every target platform](https://docs.microsoft.com/en-us/dotnet/core/rid-catalog)? What about Squirrel and update artifacts?

Self contained can be platform and runtime indepedent, this way users wont have to manually download the Core runtime. https://docs.microsoft.com/en-us/dotnet/core/deploying/deploy-with-cli

Otherwise for users who do have Core runtime or are on Win 7/8 will not be able to use it without manually go to website to install. 

> * The zip file is just the release folder compressed. Wox did that on every build, I saw no reason to remove it...

OK.

Reply:
> What about Squirrel and update artifacts?

Squirrel update should be fine as long as these three is published in Releases:
Output\Packages\Flow-Launcher-1.0.0-full.nupkg
Output\Packages\Flow-Launcher-v1.0.0.exe
Output\Packages\RELEASES

Reply:
> 
> 
> Downloaded and installed the artifact([5ed5d52](https://github.com/Flow-Launcher/Flow.Launcher/commit/5ed5d52733540c59e163f4dd2197cb7b344a25d9)), getting this error and flow doesnt start:
> "
> Description: A .NET Core application failed.
> Application: Flow.Launcher.exe
> Path: C:\Users\Blah\AppData\Local\Flow\app-1.0.0\Flow.Launcher.exe
> Message: Error:
> An assembly specified in the application dependencies manifest (Flow.Launcher.deps.json) was not found:
> package: 'PropertyChanged.Fody', version: '3.2.8'
> path: 'lib/netstandard1.0/PropertyChanged.dll'
> "

Probably because of:
```
<PrivateAssets>all</PrivateAssets>
<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
```

But from a quick try, I could not reproduce. Will look into it tomorrow...